cs/test/core/Core.csproj

```diff
@@ -54,6 +54,7 @@
     <Compile Include="SerializerGeneratorFactoryTests.cs" />
     <Compile Include="StreamTests.cs" />
     <Compile Include="Structs.cs" />
+    <Compile Include="ConvertTests.cs" />
```

Comment:
Minor: this list is alpha sorted (or vaguely alpha sorted), so can we move ConvertTests up?

Reply:
Done.